package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "",
+  "name": "template",
```

Comment:
plz change name to empty

Reply:
[bbf4b80](https://github.com/ysm-dev/template/pull/121/commits/bbf4b80b67cf5e249b857d55446ea8be8ded0b2d)